app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -127,19 +126,21 @@ object TestHelper {
         }
     }
 
-    // Remove test file from the device Downloads folder
-    @Suppress("Deprecation")
-    fun deleteDownloadFromStorage(fileName: String) {
-        runBlocking {
-            val downloadedFile = File(
-                Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS),
-                fileName
-            )
+    // Remove test file from Google Photos (AOSP) on Firebase
+    fun deleteDownloadFromStorage() {
+        val deleteButton = mDevice.findObject(UiSelector().resourceId("$GOOGLE_APPS_PHOTOS:id/trash"))
+        deleteButton.click()
```

Comment:
@AaronMT This method is meant to delete any file type from storage, regardless of where it is supposed to open. That's because when we are retrying a test the duplicate file will have the filename fail the test (because it will be called "filename(1)).